Fix `plot_sorting_summary` widget

alejoe91 opened the pull request:
Fix issue introduces by #3616

zm711 reviewed (commented):
Where is the switch to concat? I'm just trying to understand :)

alejoe91 commented:
Changed the title! I thought it was a different error, it turns out it was only this: https://github.com/SpikeInterface/spikeinterface/pull/3627/files#diff-66327b100012a00ce020c83d3a3ea191bbb820a0ce3ac02d0328387d14350051R319

Also fixed some tests due to the new kwargs

zm711 commented on src/spikeinterface/widgets/tests/test_widgets.py:
```diff
@@ -531,26 +533,37 @@ def test_plot_sorting_summary(self):
                     sw.plot_sorting_summary(
                         self.sorting_analyzer_sparse,
-                        unit_table_properties=["missing_property"],
+                        displayed_unit_properties=["missing_property"],
```
is there a reason we changed the naming? this is a breaking change? I never use this argument but do others?

samuelgarcia replied:
I think Alessio wants to test if a properties is not present. We decided to not raise an error make user life easier.

alejoe91 replied:
@samuelgarcia I think he means the argument!!!

samuelgarcia replied:
I did the change in another PR.
I prefer this one which more easy to understand.
It is back compatible.
Few users I think were using it (only for sorting view).
Now it will be usable with spikeinterface-gui.

zm711 commented on src/spikeinterface/widgets/utils.py:
```diff
@@ -316,7 +316,7 @@ def make_units_table_from_analyzer(
 
     if analyzer.get_extension("template_metrics") is not None:
-        all_df = analyzer.get_extension("template_metrics").get_data()
+        df = analyzer.get_extension("template_metrics").get_data()
```
Nice catch. I see what you mean with the append issue :)

samuelgarcia replied:
oups!!
sorry

zm711 commented on src/spikeinterface/widgets/utils_sortingview.py:
```diff
@@ -72,15 +72,15 @@ def generate_unit_table_view(
 
         ut_columns = []
         for col in unit_properties:
-            if col not in units_tables.columns:
```
I don't know this super well, but why can this be removed?

alejoe91 replied:
Because of line 77-78 :) we already filter the dataframe

samuelgarcia replied:
@alejoe91  : we decided to not raise errors when a prop is not in.
no ?

samuelgarcia replied:
yep

zm711 reviewed (commented):
couple comments :)

samuelgarcia commented:
Cool thanks camarade.
